### PR TITLE
chore: banner for `namirialid` down

### PR DESCRIPTION
**Pull request by shadowsheep1**
## Short description
Include a summary of the changes.

## List of changes proposed in this pull request
- Feature A
- Feature B

## How to test
Describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### CODE REVIEW

File: status/backend.json [hunk 1]
```diff
@@ -210,11 +210,11 @@
       }
     },
     "login": {
-      "is_visible": false,
+      "is_visible": true,
       "level": "warning",
       "message": {
-        "it-IT": "Se hai problemi con l'app, riprova tra qualche minuto. Siamo al lavoro per risolvere il problema.",
-        "en-EN": "If you are having problems with the app, please try again in a few minutes. We are working to fix the issue."
+        "it-IT": "L’accesso con NamirialID potrebbe non essere disponibile. Siamo al lavoro per risolvere il problema.",
```
Comment (thisisjp) [inline status/backend.json:216]:
```suggestion
        "it-IT": "L’accesso con Namirial ID non è disponibile. Riprova più tardi.",
```

File: status/backend.json [hunk 2]
```diff
@@ -210,11 +210,11 @@
...
+        "en-EN": "Login with NamirialID might not be available. We are working to solve the issue."
```
Comment (thisisjp) [inline status/backend.json:217]:
```suggestion
        "en-EN": "Login with Namirial ID is currently unavailable. Try again later."
```

**Review by thisisjp — CHANGES_REQUESTED** [review]
It's not an issue caused by us ;)

File: status/backend.json [hunk 3]
```diff
@@ -210,11 +210,11 @@
       }
     },
     "login": {
-      "is_visible": false,
+      "is_visible": true,
       "level": "warning",
       "message": {
-        "it-IT": "Se hai problemi con l'app, riprova tra qualche minuto. Siamo al lavoro per risolvere il problema.",
-        "en-EN": "If you are having problems with the app, please try again in a few minutes. We are working to fix the issue."
+        "it-IT": "L’accesso con Namirial ID non è disponibile. Riprova più tardi.",
```
Comment (thisisjp) [inline status/backend.json:216]:
```suggestion
        "it-IT": "L’accesso con Namirial ID non è al momento disponibile. Riprova più tardi.",
```

**Review by thisisjp — APPROVED** [review]

**Review by Ladirico — APPROVED** [review]
LGTM!

**Pull request merged**